system-int-test/src/cmr/system_int_test/utils/ingest_util.clj

```diff
@@ -304,6 +320,56 @@
          params (merge params (when accept-format {:accept accept-format}))]
      (parse-ingest-response (client/request params) options))))
 
+(defn ingest-variable
+  "Ingest a variable."
+  ([variable]
+   (ingest-variable variable {}))
+  ([variable options]
+   (let [{:keys [token client-id user-id validate-keywords validate-umm-var cmr-request-id]} options
+         accept-format (:accept-format options)
+         headers (util/remove-nil-keys {"Cmr-Validate-Keywords" validate-keywords
```

Comment:
Similar comment validate keywords is specific to UMM-C, Validate-umm-var is probably unnecessary and should only be added if we have a good reason.